safeeyes/safeeyes/config/lang/hi.json

```diff
@@ -0,0 +1,46 @@
+{
+    "meta_info": {
+        "language_name": "हिंदी",
+        "language_name_en": "Hindi"
+    },
+    "exercises": {
+        "short_break_close_eyes": "आँखों को ज़ोर से बंद करें",
+        "short_break_roll_eyes": "आँखों को घुमाएं",
+        "short_break_rotate_clockwise": "आँखों को घड़ी की दिशा में घुमाएं",
+        "short_break_rotate_counter_clockwise": "आँखों को घड़ी से उल्टी दिशा में घुमाएं",
+        "short_break_blink": "ऑंखें झपकायें",
+        "short_break_focus_far_distance": "आँखों को दूर किसी वस्तु पे टिकाएं",
+        "short_break_drink_water": "पानी पियें",
+        "long_break_walk": "थोड़ा टहलें",
+        "long_break_lean_back": "पीछे हट कर आराम करें"
+    },
+    "messages": {
+        "ready_for_a_break": "{} पलों में आराम",
+        "disabled_until_restart": "अगले आरंभ तक बंद",
+        "disabled_until_x": "{} तक बंद",
+        "next_break_at": "अगला आराम {} में"
+    },
+    "ui_controls": {
+        "skip": "अभी नहीं",
+        "short_break_duration": "छोटे आराम की अवधि (पलों में)",
+        "long_break_duration": "लंबे आराम की अवधि (पलों में)",
+        "interval_between_two_breaks": "दो आराम अवधियों में अंतर (मिन्टों में)",
+        "no_of_short_breaks_between_two_long_breaks": "दो बड़े आराम के बीच कितने छोटे आराम",
+        "time_to_prepare_for_break": "कितने पलों पहले बताएं",
```

Comment:
Hi @Trion129.

Thank you for your contribution.

`"time_to_prepare_for_break": "कितने पलों पहले बताएं",
"idle_time": "कम से कम कितने मिन्टों का अंतर बीच रोकने पर",`

Google Translate does not translate these two lines properly. I know Google Translate is not a reliable tool but I use it to double-check the translations to avoid human errors like typos. Could you please verify these two lines. If no changes are required, please let me know. Otherwise, please commit a fix.